go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/rudderlabs/rudder-go-kit
 
 go 1.22.5
 
+replace github.com/gocql/gocql => github.com/scylladb/gocql v1.14.2
```

Comment:
why are we pinning this version?

Reply:
I thought this was a good practice to pin the version as we are doing the same in rudder-server for the replace block

Reply:
Also have this error `replacement module without version must be directory path (rooted or starting with . or ..)` if I remove the version

Reply:
I get it:

```go
	module declares its path as: github.com/gocql/gocql
	        but was required as: github.com/scylladb/gocql
```

So we can't do much. The sensible way is to `replace` with `latest` but then a `go mod tidy` would replace `latest` with the version. And that's what you got now so it's cool.

```go
replace github.com/gocql/gocql => github.com/scylladb/gocql latest
```

Reply:
Basically they forked without changing the path:

https://github.com/scylladb/gocql/blob/master/go.mod

```go
module github.com/gocql/gocql

require (
	github.com/bitly/go-hostpool v0.0.0-20171023180738-a3a6125de932 // indirect
	github.com/bmizerany/assert v0.0.0-20160611221934-b7ed37b82869 // indirect
	github.com/golang/snappy v0.0.3
	github.com/google/go-cmp v0.4.0
	github.com/hailocab/go-hostpool v0.0.0-20160125115350-e80d13ce29ed
	github.com/kr/pretty v0.1.0 // indirect
	github.com/stretchr/testify v1.3.0 // indirect
	golang.org/x/net v0.0.0-20220526153639-5463443f8c37
	gopkg.in/inf.v0 v0.9.1
	sigs.k8s.io/yaml v1.3.0
)

retract (
    v1.8.0  // tag from kiwicom/gocql added by mistake to scylladb/gocql
    v1.8.1  // tag from kiwicom/gocql added by mistake to scylladb/gocql
    v1.9.0  // tag from kiwicom/gocql added by mistake to scylladb/gocql
    v1.10.0 // tag from kiwicom/gocql added by mistake to scylladb/gocql
)

go 1.13
```